cookbooks/bcpc/templates/default/carbon/carbon.conf.erb

```diff
@@ -48,9 +48,9 @@ MIN_RESET_INTERVAL=121
 METRIC_CLIENT_IDLE_TIMEOUT = <%="#{node[:bcpc][:graphite][:carbon][:relay][:idle_timeout]}"%> 
 
 [aggregator]
-LINE_RECEIVER_INTERFACE = <%="#{node[:bcpc][:graphite][:ip]}"%>
+LINE_RECEIVER_INTERFACE = <%="#{node[:bcpc][:management][:ip]}"%>
```

Comment:
Why change these here and not the other places they are defined? The use has been if we separate out a Hadoop or Kafka cluster from its Graphite cluster we can override node[:bcpc][:graphite][:ip], node[jmxtrans][:graphite][:host] and node[:chef_client][:handler][:graphite][:host].